CONTRIBUTING.md

```diff
@@ -17,8 +17,8 @@ poetry install
 
 ## Code style
 
-This project uses [PEP8] Style Guide for Python Code.  
-This project loves sorted imports.  
```

Comment:
:x: 
the two white spaces were there for a reason. they are line breaks in markdown.
please revert!